webapi/src/main/scala/org/knora/webapi/responders/EntityAndClassIriService.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.responders
+import akka.actor.ActorRef
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.scalalogging.LazyLogging
+import zio.ZLayer
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import dsp.errors.BadRequestException
+import dsp.errors.DuplicateValueException
+import org.knora.webapi.IRI
+import org.knora.webapi.messages.SmartIri
+import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.messages.store.triplestoremessages.SparqlSelectRequest
+import org.knora.webapi.messages.util.rdf.SparqlSelectResult
+
+final case class EntityAndClassIriService(
```

Comment:
Could you add a docstring indicating what this service's responsibility is / should be?

Reply:
Happy to have some suggestions. For the time being this is simply all methods extracted out of the abstract base class responder. The name suggests it has already too many responsibilities.

Reply:
I agree that this class has too many responsibilities (and I know that this is not your fault - you've just copied it). How about adding a short description like "This service handles checking of ontology entites and the creation of entity IRIs" and adding a TODO to refactor this into several classes and separate responsibilities?